src/apps/components/AppPermissionsDialog/messages.ts

```diff
@@ -0,0 +1,78 @@
+import { defineMessages } from "react-intl";
+
+const confirmation = defineMessages({
+  summaryText: {
+    defaultMessage: "You are going to",
+    id: "PlAdWI",
+  },
+  addPermissions: {
+    defaultMessage: "Add following permissions",
+    id: "oboeOT",
+  },
+  removePermissions: {
+    defaultMessage: "Remove following permissions:",
+    id: "SceSNp",
+  },
+  backButton: {
+    defaultMessage: "Go back",
+    id: "orvpWh",
+  },
+  confirmButton: {
+    defaultMessage: "I know what I'm doing - confirm",
+    id: "cS1wAx",
+  },
+});
+
+const permissionsPicker = defineMessages({
+  closeButton: {
+    defaultMessage: "Close",
+    id: "rbrahO",
+  },
+  saveButton: {
+    defaultMessage: "Save",
+    id: "jvo0vs",
+  },
+});
```

Comment:
**nitpick**: `buttonMessages` could be used from `src/intl.ts`
"Save" is there already, "Close" has to be added but this is common string that will be reused